src/app/home-foot-note/home-foot-note.component.html

```diff
@@ -26,8 +26,8 @@ <h3 class="documentation">Using Dockstore</h3>
             </a>
           </li>
           <li>
-            <a target="_blank" href="https://docs.dockstore.org/docs/publisher-tutorials/advanced-features/">
-              Advanced Features
+            <a target="_blank" href="/sitemap">
```

Comment:
Anything that is on the site should be a routerLink

Reply:
I thought so to at first, but because it's `target="_blank"`, I thought the whole SPA would get loaded on the new tab anyway, so it doesn't matter.

Reply:
ah, i missed that

Reply:
...unless it prevents it from opening in new tab

Reply:
It seems to work locally